src/rviz/TSRMarker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <aikido/rviz/TSRMarker.hpp>
+
+using dart::dynamics::SimpleFrame;
+
+namespace aikido {
+namespace rviz {
+
+TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> tsrFrames)
```

Comment:
I disagree! 😬 I prefer to leave copy versus move up to the caller (i.e. leave the code as it is).

I don't feel particularly strongly either way, so let's leave it up to @gilwoolee to decide. 😄